tests/base/pom.xml

```diff
@@ -53,10 +59,6 @@
             <groupId>com.microsoft.sqlserver</groupId>
```

Comment:
@stianst Is it necessary to keep this jdbc driver and oracle just delete?

Reply:
I think we should keep both, since Keycloak on Quarkus has support for Postgres, Myql and MariaDB